form/Rating.js

```diff
@@ -68,16 +78,31 @@ define([
 		_onMouse: function(evt){
 			// summary:
 			//		Called when mouse is moved over one of the stars
-			var hoverValue = +domAttr.get(evt.target, "value");
+			var hoverValue = +domAttr.get(evt.target.querySelector('input'), "value");
 			this._renderStars(hoverValue, true);
 			this.onMouseOver(evt, hoverValue);
 		},
 
+		_onClick: function(evt) {
+			if (evt.target.tagName === 'LABEL') {
+				var clickedValue = +domAttr.get(evt.target.querySelector('input'), "value");
+				// for backwards compatibility
```

Comment:
backwards compatibility with previous versions of Dojo or browsers or ?

Reply:
For previous versions of Dojo, because there used to be a `value` attribute directly on the `<li>` tag

Reply:
That's fine, the comment should say that so we remember what type of compatibility we were trying to preserve. :)